src/modules/components/proposal/proposalActions/proposalActionsAction/proposalActionsAction.tsx

```diff
@@ -27,14 +31,35 @@ export interface IProposalActionsActionProps extends IWeb3ComponentProps {
 export const ProposalActionsAction: React.FC<IProposalActionsActionProps> = (props) => {
     const { action, index, name, customComponent, ...web3Props } = props;
 
+    const { copy } = useOdsModulesContext();
+
+    const contentRef = useRef<HTMLDivElement>(null);
+    const itemRef = useRef<HTMLDivElement>(null);
+
     const ActionComponent = customComponent ?? proposalActionsUtils.getActionComponent(action);
 
-    const { copy } = useOdsModulesContext();
+    const [dropdownValue, setDropdownValue] = useState(ActionComponent ? 'basic' : 'decoded');
```

Comment:
Isn't this this the qualifier for disabling the whole action item in the accordion though?

Reply:
Right, we should also update the accordion logic. We used to disable it because we didn't have the raw view, as we can now display the raw data we can remove the `disabled` logic on the accordion